### PR TITLE
Base + Userland: Enhance bt command to dump all threads in a process, add a man page

### PR DESCRIPTION
**Pull request by bgianfo**
-    __bt: Fix generation of click-able links for source files.__

     When the default build location was moved from /Build to the new
     architecture specific directory, /Build/i686, this code broke.
     All file names are now path relative one additional level up.

     So continue the hack, and introduce another dummy directory to
     make the relative paths resolve correctly.

-   __bt: Enumerate all threads when symbolizing stacks in the bt utility__

    Enumerate tid's from /proc/{pid}/stacks/ and use it to print the
    backtrace for all active threads in the specified process.

-    __Base: Add a man page for the 'bt' command.__

### CODE REVIEW

File: Userland/Utilities/bt.cpp [hunk 1]
```diff
@@ -54,31 +55,42 @@ int main(int argc, char** argv)
...
+    Core::DirIterator iterator(String::formatted("/proc/{}/stacks", pid), Core::DirIterator::SkipDots);
+    if (iterator.has_error()) {
+        warnln("Error: pid '{}' doesn't appear to exist.", pid);
+        return 1;
+    }
+
+    while (iterator.has_next()) {
+        pid_t tid = iterator.next_path().to_int().value();
+        outln("tid: {}", tid);
+        // FIXME: Support multiple threads in the same process!
```
Comment (bgianfo) [inline Userland/Utilities/bt.cpp:67]: whoops

File: Base/usr/share/man/man1/bt.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
+## Name
+
+bt - View the backtrace of the specified process id
```
Comment (linusg) [inline Base/usr/share/man/man1/bt.md:3]:
```suggestion
bt - view the backtrace of the specified process
```

(lowercase just to match existing pattern)

**Review by linusg — COMMENTED** [review]
tiny nitpick for the manpage commit :)

File: Base/usr/share/man/man1/bt.md (hunk 2 above)
Comment (bgianfo) [reply Base/usr/share/man/man1/bt.md]: Fixed, thanks!

**Pull request merged**